mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
+msgstr ""
+"No ha sido posible conectar con el depurador de Python.\n"
+"\n"
 
-#: mu/modes/debugger.py:270
+#: mu/modes/debugger.py:308
 msgid "Debugger info: {}"
-msgstr "DEBUGGER INFO: {}"
+msgstr "Información del depurador: {}"
 
-#: mu/modes/debugger.py:276
+#: mu/modes/debugger.py:314
 msgid "Debugger warning: {}"
-msgstr "DEBUGGER WARNING: {}"
+msgstr "Advertencia del depurador: {}"
 
-#: mu/modes/debugger.py:283
+#: mu/modes/debugger.py:321
 msgid "Debugger error: {}"
-msgstr "DEBUGGER ERROR: {}"
+msgstr "Error del depurador: {}"
 
-#: mu/modes/microbit.py:34
+#: mu/modes/microbit.py:168
 msgid "BBC micro:bit"
-msgstr "BBC MICRO:BIT"
+msgstr "BBC micro:bit"
 
-#: mu/modes/microbit.py:35
+#: mu/modes/microbit.py:169
 msgid "Write MicroPython for the BBC micro:bit."
-msgstr "WRITE MICROPYTHON FOR THE BBC MICRO:BIT"
+msgstr "Escribe MicroPython para el BBC micro:bit."
 
-#: mu/modes/microbit.py:53
+#: mu/modes/microbit.py:190
+#, fuzzy
 msgid "Flash"
-msgstr "FLASH"
+msgstr "Flash"
 
-#: mu/modes/microbit.py:54
+#: mu/modes/microbit.py:191
 msgid "Flash your code onto the micro:bit."
-msgstr "FLASH YOUR CODE ONTO THE MICRO:BIT"
+msgstr "Programa tu código en el micro:bit."
 
-#: mu/modes/microbit.py:59
+#: mu/modes/microbit.py:197
 msgid "Files"
-msgstr "FILES"
+msgstr "Archivos"
 
-#: mu/modes/microbit.py:60
+#: mu/modes/microbit.py:198
 msgid "Access the file system on the micro:bit."
-msgstr "ACCESS THE FILE SYSTEM ON THE MICRO:BIT"
+msgstr "Accede al sistema de archivos en el micro:bit."
 
-#: mu/modes/microbit.py:66
+#: mu/modes/microbit.py:204 mu/modes/python3.py:127
+msgid "REPL"
+msgstr "REPL"
+
+#: mu/modes/microbit.py:205
 msgid "Use the REPL to live-code on the micro:bit."
-msgstr "USE THE REPL TO LIVE-CODE ON THE MICRO:BIT"
+msgstr "Use el REPL para programar en vivo en el micro:bit."
 
-#: mu/modes/microbit.py:94
+#: mu/modes/microbit.py:255
 msgid "Unable to flash \"{}\""
-msgstr "UNABLE TO FLASH \"{}\""
+msgstr "No se puede programar \"{}\""
+
+#: mu/modes/microbit.py:265
+msgid "Problem with script"
+msgstr "Problema con el script"
+
+#: mu/modes/microbit.py:266
+msgid "{} [{}:{}]"
+msgstr "{} [{}: {}]"
 
-#: mu/modes/microbit.py:95
+#: mu/modes/microbit.py:276
+msgid "Our minifier tried but your script is too long!"
+msgstr "¡Nuestro minificador lo intentó pero tu script es demasiado largo!"
+
+#: mu/modes/microbit.py:281
+msgid "Your script is too long and the minifier isn't available"
+msgstr "Tu script es demasiado largo y el minificador no está disponible"
+
+#: mu/modes/microbit.py:286
 msgid "Your script is too long!"
-msgstr "YOUR SCRIPT IS TOO LONG!"
+msgstr "¡Tu script es demasiado largo!"
 
-#: mu/modes/microbit.py:122
+#: mu/modes/microbit.py:357
 msgid "Flashing \"{}\" onto the micro:bit."
-msgstr "FLASHING \"{}\" ONTO THE MICROBIT"
+msgstr "Programando \"{}\" en el micro:bit."
 
-#: mu/modes/microbit.py:124
+#: mu/modes/microbit.py:359
 msgid " Runtime: {}"
-msgstr "RUNTIME: {}"
+msgstr " Interpretador: {}"
```

Comment:
In this case it is referring to the MicroPython "runtime", so while it is more than an interpreter, I thought it would be easier to understand using that word.

Reply:
Perfect.